content/en/docs/guides/custom-backend-metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Custom Backend Metrics
+description : >-
+  A mechanism in gRPC library that allows users to inject custom metrics at the
+  gRPC server side (backends). These metrics are then reported to the gRPC
+  clients where they can be consumed by the custom load balancing algorithms to
+  make traffic distribution decisions.
+---
+
+
+### Overview
+Simple load balancing decisions can be made by taking into account local or
+global knowledge of a backend’s load, for example CPU. More sophisticated
+loadbalancing decisions are possible with application specific knowledge, e.g.
+queue depth, or by combining multiple metrics.
+
+The custom backend metrics feature exposes APIs to allow users to implement
+the metrics feedback in their LB policies.
+
+### User Cases
+The feature is mainly for advanced uses cases where a custom LB policy is used
+to route traffic more smartly to a list of backend servers to improve the
+routing performance, e.g. a weighted round robin LB policy.
+
+gRPC traditionally allows users to plug in their own load balancing policies.
+For xDS users, [custom load balancing] can be configured to select the
```

Comment:
This link makes it seem that the linked doc is xDS-specific. Can we move this to the first reference to custom load balancing policies on the previous line, which is not xDS-specific?